src/app/containers/Include/index.jsx

```diff
@@ -25,12 +25,73 @@ const decodeHTML = (str) => {
 
 const IncludeContainer = ({ html, type }) => {
   const { enabled } = useToggle('include');
+  // const [data, setData] = useState('');
+  const [includeHtml, setIncludeHtml] = useState('');
```

Comment:
A point just occured to me that all the logic in useEffects will execute on the client and not on the server; this means that this PR will disable serverside rendering of includes in its current form. I believe there is a way of telling if you are on the server or not (though I’ve never done this myself). 

So I think...
- We want to dangerously insert the html unchanged on the server and disable all the useEffect logic
- On the client we want to initially not render the include as you have done here and perform your useEffect logic to extract and relocate the scripts